car_showroom-2/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+<navbar />
```

Comment:
- use one main div in template and do all html code in that div for good practice
- use code formatter for code formatting